docs/SignedAPKAndroid.md

```diff
@@ -23,6 +23,8 @@ _Note: Remember to keep your keystore file private and never commit it to versio
 
 ### Setting up gradle variables
 
+_Note: This is a global way that will work for any platform, if you are not keen on storing your passwords in plaintext and you are running OSX, you can also [store your credentials in the Keychain Access app](https://pilloxa.gitlab.io/posts/safer-passwords-in-gradle/). Then you can skip the two last rows in `~/.gradle/gradle.properties`._ 
```

Comment:
Actually it won't work for Windows. So let's remove the text `This is a global way that will work for any platform`